frontend/src/views/app-store/detail/install/index.vue

```diff
@@ -173,7 +173,7 @@ const installData = ref<InstallProps>({
 });
 const open = ref(false);
 const rules = ref<FormRules>({
-    name: [Rules.appName],
+    name: [Rules.paramSimple],
     params: [],
     version: [Rules.requiredSelect],
     containerName: [Rules.containerName],
```

Comment:
The difference in this code snippet is that the `appName` rule from `"Rules"` has been replaced with `"paramSimple"`. It's unclear what these rules are defined as, but replacing one with another could have different behaviors or constraints depending on their implementation. Ensure both "appName" and "paramSimple" rules behave similarly to avoid unexpected behavior in the application.